itest/rust/src/object_tests/init_level_test.rs

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use crate::framework::itest;
+use godot::{
+    init::InitLevel,
+    obj::NewAlloc,
+    register::{godot_api, GodotClass},
+};
+use std::sync::atomic::{AtomicBool, Ordering};
+
+static HAS_RUN: AtomicBool = AtomicBool::new(false);
+
+#[derive(GodotClass)]
+#[class(base = Object, init)]
+struct SomeObject {}
+
+#[godot_api]
+impl SomeObject {
+    #[func]
+    pub fn trigger_has_run(&self) {
+        HAS_RUN.store(true, Ordering::Release);
+    }
```

Comment:
Name sounds like "trigger has run" i.e. boolean. Maybe `mark_called` or so would be clearer, in the sense that you just want to mark it as called?